apps/package.json

```diff
@@ -42,7 +42,7 @@
   },
   "devDependencies": {
     "@cdo/apps": "file:src",
-    "@code-dot-org/blockly": "1.8.2",
+    "@code-dot-org/blockly": "1.8.2-add-anglehelper-options",
```

Comment:
Did you mean to publish this as a non-feature version before merging this PR?

Reply:
oh shoot, I spaced! Thanks